lib/serverless-cli.js

```diff
@@ -2,14 +2,26 @@
 
 const version = require('./../package.json').version;
 const program = require('commander');
-const { exec } = require('./exec');
+const { create } = require('./actions/create');
+const { addPlugins } = require('./actions/plugins');
 
+// Update version
+program.version(version);
+
+// Create
 program
-  .version(version)
+  .command('create <template>')
   .option('-i, --init', 'Initialize a package.json with all defaults')
-  .parse(process.argv);
+  .option('--verbose', 'Verbose, show info logs in addition to errors')
+  .action(create);
+
+// Plugins
+program
+  .command('plugins <plugin> [otherPlugins...]')
+  .option('-a, --add', 'Add serverless plugin(s)')
+  .option('-r, --remove', 'Remove serverless plugin(s)')
+  .option('--verbose', 'Verbose, show info logs in addition to errors')
```

Comment:
Might need to later combine the verbose functionality, but later as not a dire concern and more obvious when in one location for now.